efficientdet/det_model_fn.py

```diff
@@ -460,6 +475,8 @@ def metric_fn(**kwargs):
                   nms_configs['max_output_size'],
               ], tf.float32)
           detections_bs.append(detections)
+        detections_bs = postprocess.transform_detections(
```

Comment:
This detection_bs does not appear to be used elsewhere? so why bother calculating it? Also, shouldn't the detection_bs in line 497 also be transformed if this is used?

Reply:
used in eval_metric

Reply:
where? pycharm says this variable is not used, I also don't see it passed anywhere?

Reply:
line 503

Reply:
ohw, my bad, I edited out the coco stuff in my version. Thanks!